sdk/core/azure-core-test/pom.xml

```diff
@@ -44,37 +44,45 @@
     <dependency>
       <groupId>io.projectreactor</groupId>
       <artifactId>reactor-test</artifactId>
+      <version>3.3.0.RELEASE</version> <!-- {x-version-update;io.projectreactor:reactor-test;external_dependency} -->
     </dependency>
```

Comment:
@srnagar @hemanttanwar @conniey Should this be a test dependency?

Reply:
@JonathanGiles This may be rehashing an old discussion around whether Azure Core Test should exist as a standalone package or be part or Azure Core and generated as a test JAR.

Reply:
I would like azure-core-test to be removed, but it is easier said than done. It is difficult at present due to JDK9+ modules and the visibility of some classes that we couldn't quite resolved.